cosmo_tester/test_suites/manager_of_managers/regional_cluster.py

```diff
@@ -0,0 +1,974 @@
+########
+# Copyright (c) 2018 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+import json
+
+from cloudify_cli.constants import DEFAULT_TENANT_NAME
+
+from cosmo_tester.framework import util
+from cosmo_tester.framework.examples import AbstractExample
+
+from . import constants
+
+
+class AbstractRegionalCluster(AbstractExample):
+    # TODO: Update this to use jenkins user.
+    REPOSITORY_URL = 'https://github.com/' \
+                     'cloudify-cosmo/cloudify-spire-plugin.git'
+    TRANSFER_AGENTS = None
+
+    def __init__(self, *args, **kwargs):
+        kwargs['branch'] = 'master'
```

Comment:
I think there is a version env param of the tested version

Reply:
yes, i will remove this.